lib/v2/weibo/search/template/digest.art

```diff
@@ -0,0 +1,10 @@
+<div class="quoted">
+    <a style="text-decoration: none;" href="{{ author.link }}">{{ author.name }}</a>
+    <span><a href="{{link }}"> | {{ postinfo }} </a></span>
+</div>
+<div class="content">
+{{@ msg }}
+</div>
+<div>
+------
+</div>
```

Comment:
You can use `<hr>`

Reply:
Thanks, Tony, and actually `<hr>` was used in very beginning, and I turn to used plain `'---' ` eventually, just because of in Weibo items, normally it's just quite short status with characters <140, i.e. 1 or 1.5 lines around. After using `hr` there, it makes the page (in most rss readers) much more 'fragmented' and not comfortable (at least in my eyes.), so I would suggest to still stay in using the plain text, which can make each entry looks really like one 'digest style'... sounds OK?